cursive-core/src/buffer.rs

```diff
@@ -183,10 +183,17 @@ impl PrintBuffer {
         // TODO: Use some WithWidth(&str, usize) to not re-compute width a thousand times
         for g in text.graphemes(true) {
             let width = g.width();
+            //if (width == 0) || ("\0" == g) {
             if width == 0 {
                 continue;
             }
-            self.set_cell(pos, g, CellWidth::from_usize(width), style);
+            if "\0" == g {
```

Comment:
We might want to do this for all control characters, not just \0.

Reply:
Sounds good but I don't think I'm equipped to do this myself, and even do it close to properly, that's why I thought it's best to leave this for you to do(better,faster and without wasting your time(reviewing my coding mistakes) too). This was just a proof of concept commit which I can easily undo. What do you think?

Here's the list of all(?) control characters: https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=\p{Cc} taken from https://github.com/unicode-rs/unicode-width/pull/45/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L59

Kind of interesting to see how they do it in `unicode-width`, for `&str` and for `char`:

https://github.com/unicode-rs/unicode-width/commit/4efb1803faa054f1bea3c0457275ad3c8610170b#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R201-R228

https://github.com/unicode-rs/unicode-width/commit/4efb1803faa054f1bea3c0457275ad3c8610170b#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R130-R149